### DISCUSSION TITLE
Here I am again with modcharting.

### DISCUSSION THREAD

**Discussion by TheCapitalist** [KadeArchive/Kade-Engine#1003] [Q&A] [answered]
Basically I wanted to do a flashbang effect, and I dont seem to be able to use tweenFadeOut to make it dissapear, anyone knows how to fix the tweenFadeOut or make the same effect?

**Comment by AmberHaxe** [comment, marked as answer]
in tween fade out, the first value is the actor alpha, so if you want the white background to tween in you'd set it to 1, to turn it off, you'd set it to 0.

ex.  

if curStep == 10 then
tweenFadeOut('WhiteBG' ,1, 2)
end

if curStep == 20 then
tweenFadeOut('WhiteBG' ,0,2)
end

**Reply by TheCapitalist** [reply]
That's the very first thing I did, I have no idea why it doesnt work

**Reply by ZenusPurity** [reply]
if curStep == 20 then
tweenFadeOut('WhiteBG' ,0,2)
end

Change this to tweenFadeIn instead of tweenFadeOut

**Reply by AmberHaxe** [reply]
> if curStep == 20 then
> tweenFadeOut('WhiteBG' ,0,2)
> end
> 
> Change this to tweenFadeIn instead of tweenFadeOut

this isn't necessary, you can still use tweenFadeOut to fade in an actor, it works perfectly fine. you can also use tweenFadeIn like you mentioned though. Again tho, not necessary.

**Reply by TheCapitalist** [reply]
again, I tried using tweenFadeIn instead of Out, still didnt work

**Reply by TheCapitalist** [reply]
and finally got it, turns out I had to first use tweenFadeIn(WhiteBG,1,2) and then use tweenFadeOut, basically to make the flash instant I set the actor alpha to 100 and then use tweenFadeIn
